### PR TITLE
Change default current round

### PR DESCRIPTION
**Pull request by shine2lay**

### CODE REVIEW

File: test/startRoundUnitTest.js [hunk 1]
```diff
@@ -45,7 +46,8 @@ contract('ROSCA startRound Unit Test', function(accounts) {
     }));
 
     it("Throws when calling startRound before roundStartTime (including round = 0)", co(function* () {
```
Comment (sihoang) [inline test/startRoundUnitTest.js]: the description and test case need to be updated. Maybe `Throws when calling startRound during round 1`?

File: test/withdrawUnitTest.js [hunk 2]
```diff
@@ -25,10 +25,10 @@ contract('ROSCA withdraw Unit Test', function(accounts) {
 
     it("Throws when calling withdraw from a non-member", co(function* () {
         yield Promise.all([
-            ethRoscaHelper.contribute(0, consts.CONTRIBUTION_SIZE),
-            ethRoscaHelper.contribute(1, consts.CONTRIBUTION_SIZE),
-            ethRoscaHelper.contribute(2, consts.CONTRIBUTION_SIZE),
-            ethRoscaHelper.contribute(3, consts.CONTRIBUTION_SIZE),
+            ethRoscaHelper.contribute(0, 2 * consts.CONTRIBUTION_SIZE),
```
Comment (sihoang) [inline test/withdrawUnitTest.js:28]: this is not related to Round 0 -> 1?

File: test/withdrawUnitTest.js [hunk 3]
```diff
@@ -44,23 +44,23 @@ contract('ROSCA withdraw Unit Test', function(accounts) {
...
-        const ACTUAL_CONTRIBUTION = consts.CONTRIBUTION_SIZE * 0.8;
+        const ACTUAL_CONTRIBUTION = consts.CONTRIBUTION_SIZE * 1.8;
+        const EXPECTED_WITHDRAWAL = consts.CONTRIBUTION_SIZE * 0.8;
 
         yield roscaHelper.contribute(0, ACTUAL_CONTRIBUTION);
 
         let result = yield roscaHelper.withdraw(0);
         let log = result.logs[0];
 
-        assert.equal(log.args.user, accounts[0], "LogContributionMade doesn't display proper user value");
```
Comment (sihoang) [inline test/withdrawUnitTest.js:54]: we actually call contribute. Should this assert pass?

Comment (shine2lay) [reply test/withdrawUnitTest.js:54]: We are only watching for LogFundsWithdrawal, it was an error on previous version. only the error string was wrong. the assert is right

File: test/withdrawUnitTest.js (hunk 2 above)
Comment (shine2lay) [reply test/withdrawUnitTest.js:28]: it is, this test originally test withdrawal in round 0 , but we are in round 1 , we need to contribute more than round 1's contribution to be able to withdraw

File: test/startRoundUnitTest.js (hunk 1 above)
Comment (shine2lay) [reply test/startRoundUnitTest.js]: i think we just need to get rid of `(including round 0)` because it goes through all the rounds

File: test/startRoundUnitTest.js [hunk 4]
```diff
@@ -18,6 +18,14 @@ contract('ROSCA startRound Unit Test', function(accounts) {
       roscaHelper = new ROSCAHelper(accounts, (yield utils.createEthROSCA()));
     }));
 
+  it("checks that round 1 can be longer one roundPeriod", co(function* () {
```
Comment (sihoang) [inline test/startRoundUnitTest.js:21]: indentation

File: test/startRoundUnitTest.js [hunk 5]
```diff
@@ -18,6 +18,14 @@ contract('ROSCA startRound Unit Test', function(accounts) {
...
+    utils.increaseTime(consts.ROUND_PERIOD_IN_SECS);
+    yield utils.assertThrows(roscaHelper.startRound(),
+      "expected calling startRound before roundStartTime to throw");
+    utils.increaseTime(consts.START_TIME_DELAY);
+    yield roscaHelper.startRound();
```
Comment (sihoang) [inline test/startRoundUnitTest.js:26]: do we need to assert the startRound succeeds?

**Review by sihoang — APPROVED** [review]
merge for code audit. Please submit the fix later.
